src/wp-includes/class-wp-theme-json.php

```diff
@@ -519,7 +521,10 @@ public function __construct( $theme_json = array(), $origin = 'theme' ) {
 		$nodes = static::get_setting_nodes( $this->theme_json );
 		foreach ( $nodes as $node ) {
 			foreach ( static::PRESETS_METADATA as $preset_metadata ) {
-				$path   = array_merge( $node['path'], $preset_metadata['path'] );
+				$path = $node['path'];
+				foreach ( $preset_metadata['path'] as $subpath ) {
+					$path[] = $subpath;
```

Comment:
Just noting that this doesn't protect against duplicate values being added to `$path`, but that's how it works currently as `array_merge()` also results in duplicates values being present.

Reply:
Right, was looking at this too, but indeed not a change in behavior, so this is fine as is.